web/src/routes/ShelterPage.tsx

```diff
@@ -20,10 +22,17 @@ const ShelterPage = ({ cityModel, cityCode, languageCode, pathname, languages }:
   const { shelterId } = useParams()
   const { viewportSmall } = useWindowDimensions()
   const { t } = useTranslation('shelter')
+  const [filter, setFilter] = useState<FilterProps>({ beds: null, pets: null })
```

Comment:
This default should be fine, then you don't need to check whether params are defined in the endpoint. At least it makes no sense to search for less than one bed.
```suggestion
  const [filter, setFilter] = useState<FilterProps>({ beds: '1', pets: '0' })
```

Reply:
i would like to keep the placeholder in the input field --> rejected